core-bundle/src/Command/UserCreateCommand.php

```diff
@@ -165,12 +165,7 @@ protected function interact(InputInterface $input, OutputInterface $output): voi
         }
 
         if (false === $input->getOption('admin') && ($options = $this->getGroups()) && 0 !== \count($options)) {
-            $answer = $this->askMultipleChoice(
-                'Assign which groups to the user (select multiple comma-separated)?',
-                $options,
-                $input,
-                $output
-            );
+            $answer = $this->askMultipleChoice($options, $input, $output);
```

Comment:
If this command needs another multiple choice question in the future, this change basically has to be reverted again. Otherwise the method needs to be renamed to `askUserGroup`, since this method is now specifically _only_ to ask about the user groups after this change.

Reply:
> If this command needs another multiple choice question in the future, this change basically has to be reverted again.

Yes. Is this a problem?

Reply:
No, but then I vote for changing the name of the method to reflect what it actually does.

Reply:
Reverted in 3fa8d4e0e56b089d8476fab4976d0d13729d7f2d.